src/app/(home)/mypage/components/ProfileInput/ProfileInput.tsx

```
@@ -1,8 +1,14 @@
-import { ChangeEvent, useEffect, useState } from 'react';
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react';
```

ESLint

[error] 'FormEvent' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)